### ISSUE TITLE
Prepare NHD region 6 RWD data

### ISSUE THREAD

**Issue by mmcfarland** [WikiWatershed/rapid-watershed-delineation#30]
Preprocessed data required for running the NHD RWD version from #29 is located here: http://hydrology.usu.edu/mmw2/NHDplusRegion6/

This data needs to be copied and extracted to the RWD worker instance like the DRB data currently is.  If the code runs and executes successfully, we'll need to make arrangements for a method to scale up the data size.

**Comment by mmcfarland** [comment]
Extracted, the two zips referenced above some out to `5.4g` and `711mb`

**Comment by kdeloach** [comment]
Copying the NHD RWD data on top of the old RWD data works without conflict we're just waiting on a newer version of TauDEM before we can utilize the NHD code.

**Comment by mmcfarland** [comment]
The target region changed, this is superseded by #34
